packages/server/graphql/types/GitLabIntegration.ts

```diff
@@ -19,7 +19,7 @@ const GitLabIntegration = new GraphQLObjectType<any, GQLContext>({
     },
     cloudProvider: {
       description:
-        'The cloud provider the team member may choose to integrate with. Nullable based on env vars',
+        'The cloud provider the team member may choose to integrate with. Nullable based on env vars, e.g. a self-hosted instance',
```

Comment:
-1 This description adds more confusion for me, do you mean "Could be null for self hosted parabol instances" or "Could be set to null if only self hosted GitLab servers are supported"?

Reply:
Yeah, I meant it could be null for self-hosted Parabol instances, but I'm happy to remove this if it adds more confusion